### PR TITLE
LightAlloy

### PR DESCRIPTION
**Pull request by Tobias-B-Besemer**

### CODE REVIEW

**Comment by az0** [comment]
Of the several pull requests I checked, all had `xnview.xml`, which should be only in one PR.

File: pending/lightalloy.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	CleanerML written by https://github.com/Tobias-B-Besemer
+	v0.1 done on 2018-12-20
+
+-->
+<cleaner id="lightalloy">
```
Comment (az0) [inline pending/lightalloy.xml]:
```suggestion
<cleaner id="lightalloy" os="windows">
```

File: pending/xnview.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	CleanerML written by https://github.com/Tobias-B-Besemer
+	v0.1 done on 2018-12-19
+
+-->
+<cleaner id="xnview.new">
```
Comment (az0) [inline pending/xnview.xml]: This file conflicts with the other pull request https://github.com/az0/cleanerml/pull/151

File: pending/lightalloy.xml (hunk 1 above)
Comment (Tobias-B-Besemer) [reply pending/lightalloy.xml]: Done.

File: pending/xnview.xml (hunk 2 above)
Comment (Tobias-B-Besemer) [reply pending/xnview.xml]: Fixed.

**Pull request merged**